why3/Cargo.toml

```diff
@@ -12,7 +12,7 @@ itertools = "*"
 pretty = "0.11"
 indexmap = "1.2.0"
 serde = { version = "1.0", optional = true, features = ["derive"] }
-
+num = "*"
```

Comment:
Not really elegant to pull this whole thing in just to get the bit patterns of a float's components, but /shrug.